src/java.security.jgss/share/classes/sun/security/krb5/Credentials.java

```diff
@@ -528,6 +532,8 @@ public static void printDebug(Credentials c) {
     static void ensureLoaded() {
         if (OperatingSystem.isMacOS()) {
             System.loadLibrary("osxkrb5");
+        } else if (OperatingSystem.isLinux()) {
+            System.loadLibrary("linuxkrb5");
```

Comment:
Can we use the same lib name on macOS and Linux?

Reply:
We can - this was mostly just an attempt to avoid changing MacOS for the sake of a library name - I also wasn't sure if this might lead to other packaging changes/issues for people who package JVMs for MacOS?  My original approach had a single name, so it would straightforward to make this happen.  What do you think?

Reply:
I can only speak for myself/my employer, but we would find utility in this feature being backported for sure.  I'll leave as-is for now, but happy to adjust in the future.